.github/workflows/ci-pipeline.yml

```diff
@@ -0,0 +1,78 @@
+name: CI Pipeline for DoDo
+
+on:
+  pull_request:
+    branches:
+      - develop
+      - main
+  push:
+    branches:
+      - develop
+
+jobs:
+  test_and_build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Docker Compose
+      run: |
+        docker compose version
```

Comment:
Please update the step name if you want to just print docker version.

Reply:
The step name is misleading since it’s only printing the Docker Compose version, not setting it up.